layouts/partials/site-nav.html

```diff
@@ -12,10 +12,10 @@ <h1 class="dim f3 lh-solid ml0-ns mr0 mr4-l mv0 pl3 pl4-ns">
       {{ range .Site.Menus.global }}
         <li class="f5 dib mr4" role="menuitem">
             {{/* TODO: Create an "Global" active class to show which site one is currently at */}}
-          <a href="{{ .URL }}" class="dim link{{ if $currentPage.IsMenuCurrent "global" . }} gray {{else}} light-silver{{ end }}">
-            {{ .Name }}
-              {{/* using ".Post" from the menu system to determine if we should show an icon for external links */}}
+            {{/* using ".Post" from the menu system to determine if we should show an icon for external links */}}
             {{ $post_status := printf "%s" .Post }}
+          <a href="{{ .URL }}" class="dim link{{ if $currentPage.IsMenuCurrent "global" . }} gray {{else}} light-silver{{ end }}" {{ if eq $post_status "external" }}target="_blank" rel="noopener noreferrer"{{ end }}>
```

Comment:
Adding noreferrer to external links can have a negative impact on analytics tracking. I suggest to remove that. It's good to keep noopener (which has wide browser support) - but no need for noreferrer.